db/helpers/common.js

```diff
@@ -188,7 +201,11 @@ function publishChange(inst, event, changedKeys, ignoreAttributes) {
   }
 
   if (obj[event]) {
-    pub.publish(channelName, JSON.stringify(obj));
+    if (isRoom(inst)) {
+      pubBot.publish(botChannelName, JSON.stringify(obj));
+    } else {
+      pubPerspective.publish(perspectiveChannelName, JSON.stringify(obj));
+    }
   }
```

Comment:
You probably want to make these changes to the "publishObject" method in /realTime/redisPublisher.js.